src/components/ModuleTranslator/ModuleTranslator.js

```diff
@@ -23,6 +23,7 @@ class ModuleTranslator extends React.Component {
       app: (originalModules.app || []).map(this.translateModule),
       plugin: (originalModules.plugin || []).map(this.translateModule),
       settings: (originalModules.settings || []).map(this.translateModule),
+      handler: (originalModules.handler || []).map(this.translateModule),
```

Comment:
What does this do? What kind of thing is `originalModules.handler`? A string, a function? From its use in `<Handlers>`, it _looks_ like it's the module structure -- in which case does it make any sense to translate it? (I realise this is a comment not so much on your code as it is on my own ignorance.)

Reply:
@MikeTaylor I'm not sure either. I had to add this in order to actually get the `handler` in a lower level components. I think it happensbecause the ModuleContext it's being overwritten `ModuleTranslator` for some reason.

https://github.com/folio-org/stripes-core/blob/9d33e12950b322ae2a4ca23721168d7658df3808/src/components/ModuleTranslator/ModuleTranslator.js#L41

But this is really not related to our `handler` work here. We can clean that up later.

Reply:
OK, having looked at the stripes-core code, I realise what's going on here. This is another instance, like in `findModule`, where we're just manually iterating over the different known kinds of module. So, yes, what you've done makes perfect sense; and it's open to a nicer refactoring like what you did to `findModule` but that's not _necessary_.